lib/message.rb

```diff
@@ -0,0 +1,24 @@
+require 'twilio-ruby'
+require 'time'
+
+class Message
+
+  attr_reader :time
+
+  def initialize
+    @time = Time.now + 1 * 60 * 60
```

Comment:
Without a comment, it's difficult to understand why you're adding 3600 to `Time.now`. What do you think about adding a comment here?

Is this something to do with british summer time? Does `Time` have an inbuilt way of giving you the correct time accounting for british summer time?

Reply:
Whilst looking up how to add 1 hour to a new instance of time, everything pointed to the syntax I have used. On testing, the text message does send confirming that the food will be with the customer 1 hour from the current time.

Reply:
Ah, that makes sense! Then a comment to explain that you want the text to be sent an hour after the order would help the reader understand the purpose behind this line of code 🙂